theme/less/_homepage.less

```diff
@@ -0,0 +1,59 @@
+.homepage
+{
+  grid-template-areas: 
+    "header header"
+    "main main"
+    "footer footer";
+}
+
+.homepage main article
+{
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(346px, 1fr));
+}
```

Comment:
I used this in order to be simple with widths and heights of items.

Reply:
Hmm, having a closer look, this might be not a bad idea, only thing is that we might rethink the implementation after adding heading into the `<main>` block.